tests/OmniSharp.Tests/OmniSharp.Tests.csproj

```diff
@@ -6,6 +6,8 @@
     <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>
   </PropertyGroup>
 
+  <Import Project="../../build/netfx.props" />
```

Comment:
Can these be brought in via `Directory.build.props` or `Directory.build.targets`?

Seems silly to have to add it to all project files if we could avoid it.

Reply:
I tried that but it doesn't get loaded at all then. Seems like `<Import Project="..."  />` doesn't work via those extensibility points.